usaspending_api/common/helpers/spark_helpers.py

```diff
@@ -401,6 +414,19 @@ def get_jdbc_url():
     return get_jdbc_url_from_pg_uri(CONFIG.DATABASE_URL)
 
 
+def get_broker_jdbc_url(databroker_uri: str)->str:
```

Comment:
Any reason why this isn't following the other methods? Feels like it makes more sense to just do `get_broker_jdbc_url()` that's just like `get_jdbc_url` except it's `DATA_BROKER_DATABASE_URL` and that it reuses `get_jdbc_url_from_pg_uri` as both will be postgres connection strings.

Reply:
Refactored to follow other methods